app/src/main/java/com/uni/gruppenphaseandroid/SensorReader_Light.java

```diff
@@ -46,11 +48,8 @@ public void onAccuracyChanged(Sensor arg0, int arg1) {
         public void onSensorChanged(SensorEvent event) {
 
             if (event.values[0] <= event.values[0]){
-              /*  var message = new Message();
-                message.setType(MessageType.CHEATING_LIGHT);
-                websocketClient.send(message);
-            */
-                //TODO send message to GameManager (prüft vorab ob schummeln möglich ist, wenn ja wird die Message den Server geschickt)
+                System.out.println("sensor_light");
```

Comment:
just FYI. das sollte nichts machen. um was in die console zu schreiben muss man es mit log machen (glaub ich)